app/views/pets/show.html.erb

```diff
@@ -16,13 +8,16 @@
 <div class="container">
   <div class="row">
     <div class="col-6 center">
-      <img src=<%= "#{@pet.img_url}" %> alt=<%= "#{@pet.id}" %> height="100" width="100">
-      <p>Approximate Age: <%= @pet.age %></p>
-      <p>Sex: <%= @pet.sex %></p>
-      <p>Description: <%= @pet.description %></p>
-      <p>Adoption Status: <%= @pet.adoption_status %></p>
-      <%= link_to "Update Pet", "/pets/#{@pet.id}/edit" %>
-      <%= button_to "Delete Pet", "/pets/#{@pet.id}", method: :delete %> 
+      <section id="pet-<%= @pet.id %>">
+        <img src=<%= "#{@pet.img_url}" %> alt=<%= "#{@pet.id}" %> height="100" width="100">
+        <p>Approximate Age: <%= @pet.age %></p>
+        <p>Sex: <%= @pet.sex %></p>
+        <p>Description: <%= @pet.description %></p>
+        <p>Adoption Status: <%= @pet.adoption_status %></p>
+        <%= link_to "Update Pet", "/pets/#{@pet.id}/edit" %>
+        <%= button_to "Delete Pet", "/pets/#{@pet.id}", method: :delete %>
+        <%= button_to "Add Pet to Favorites", "/favorites/#{@pet.id}", method: :patch %> 
```

Comment:
Is there a reason you use Patch instead of Put? It is my understanding that it wouldnt matter which one you used in this specific case, but wondering if you prefer one or the other.